README.md

```diff
@@ -304,6 +305,29 @@ function messageCallBack(params){
 }
 
 ```
+#### Download event Example
+
+Whenever the InAppBrowser receives or locates to a url which leads in downloading a file, the callback assgined to the "download" event is called. The parameter passed to this callback is an object with the the follwing properties
+
+- **type** _it contains the String value "download" always_
+- **url** _The url that leaded to the downloading of file. Basically, the download link of file_
```

Comment:
Is this a `http` url or a local filesystem url?

Or I guess in other words, is this file already downloaded it, or is the developer expected to implement logic of actually downloading the requested resource?

I'm unfamiliar with the `DownloadListener` API, but if it is expected for the developer to download the resource, perhaps a small XHR request example can show this. e.g:

```
var xhr = new XMLHttpRequest();
xhr.open("GET", params.url);
xhr.onload = function() {
    var content = xhr.responseText;
};
xhr.send();
```

If the file is already downloaded, perhaps a quick mention and link to the `cordova-plugin-file` for reading files.

Reply:
The url will be an http url to the file.
The downloading of the file (if required) have to be explicitly done by using the http url received in the callback to the event.